styles.css

```diff
@@ -61,19 +44,15 @@ nav {
   margin-right: 0;
 }
 
-main {
+.main {
   display: flex;
   flex-direction: column;
   align-items: center;
 }
 
 .myname {
```

Comment:
[Optional] [Highly Recommended] For styling, please consider the following changes I marked them as optional because the previous reviewer forgot to mention them.

- [ ] For the Header please make the logo and the hamburger button bigger and put more margin-top to them to match the given Figma design.
- [ ] For the technologies section, please make only two columns and wrap the third one below them like the given design (You can use flexbox to do this).
- [ ] For the social links, Please make them bigger and center them in the middle, and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170695055-6337be46-e2ac-4948-8358-9f644170ce47.png)  | ![image](https://user-images.githubusercontent.com/68796232/170694963-815075ae-3ced-42c0-ba69-6501a780e0ed.png)  |